### ISSUE TITLE
Could you please please fix this?

### ISSUE THREAD

**Issue by lightning02** [BluSunrize/AquaTweaks#5]
![567](https://cloud.githubusercontent.com/assets/11884147/15542409/f0ed90e6-2290-11e6-881b-d8ffc0343ed5.png)

AquaTweaks seems to have some problem with fresh water from TerraFirmaCraft (was using v 0.79.29.922 of tfc)
Even without Optifine the problem seems to occur..
it would be really really Amazing use aqua tweaks in a realistic enviroment as TFC.. I hope there is something that can be done for this problem :c

**Comment by BluSunrize** [comment]
Nah. TFC does a looooot of coremodding. I can't really be expected to account for that. Also, this is 1.7.10, I don't develop on that anymore.
